spec.md

```diff
@@ -120,8 +120,8 @@ The following abstract data types are available for use in attributes.
 
 This specification does not define numeric or logical types.
 
-The `Any` type is a variant type that can take the shape of either a
-`String` or a `Binary` or a `Map`. The type system is intentionally
+The `Any` type is a variant type that can take the shape of either a `String`,
+or a `Binary`, or a `Map`, or an `Integer`. The type system is intentionally
```

Comment:
@Tapppi we discussed this on today's call and people are ok with the general direction. They did, however, want to make some minor editorial changes - would you be ok with:
... take the shape of either a `Binary`, `Integer`, `Map` or `String`.
Basically, alphabetize, remove the extra "or a"s and the extra comma.